src/blockchain/node/manager/Manager.cpp

```diff
@@ -340,7 +340,7 @@ Base::Base(
         try {
             const auto boost = boost::asio::ip::make_address(addr);
 
-            if (!boost.is_v4()) {
+            if (false == boost.is_v4()) {
```

Comment:
why are we restoring this c-era nonsense? The guidelines someone uses to justify the huge effort expended for making everything const expressly warn against unnecessary comparisons with bool values.

Reply:
We Both Agreed on this as this false prevents from future errors and all merges whas picked with that change. 
What would be the benefit on not introducing it besides the disadv of future merges ?

Reply:
hold up:D
"this false prevents from future errors " - this has nothing to do with errors. It just visual thing.

Reply:
yes its visual, till then someone try to chage it. we can do false = dupa or dupa = false. but im not so expert to insist this. Justus started and we do not have any significant argument to stop it. Or maybe im missing sth, would be glad to learn sth

Reply:
With an assignment written where a boolean condition is expected, any half decent compiler from about 2005 onward issues a warning - hope you have learned something, after all.